tests/components/switch/test_command_line.py

```diff
@@ -155,3 +156,13 @@ def test_state_code(self):
 
             state = self.hass.states.get('switch.test')
             self.assertEqual(STATE_ON, state.state)
+
+
+    def test_assumed_state(self):
+       """Test with state value."""
+       self.hass = get_test_home_assistant()
+       name = 'Test'
+       oncmd = 'echo 1 > {}'
```

Comment:
``` oncmd ``` and ``` offcmd ``` are distracting from the purpose of this test.
I would recommend only creating local variables for values that are important to the test.

``` command_state = None ``` is important but is currently missing.